tests/ui/specs/TimePicker/TimePicker-specs.js

```diff
@@ -237,6 +290,61 @@ describe('TimePicker', function () {
 			await Page.spotlightLeft();
 			await browser.waitUntil(async () => await timePicker.meridiem.isFocused(), {timeout: 1500,  timeoutMsg: 'meridiem', interval: 100});
 		});
+
+		it('should move to the next picker with 5-way OK key in ar-SA locale [QWTC-2541]', async function () {
+			// Step 10-1: hover on the hour picker.
+			await Page.showPointerByKeycode();
+			await $('#timePickerDefault').moveTo({xOffset: 200, yOffset: 100});
+			// Step 10-1 Verify: Spotlight on the hour picker.
+			await expect(await timePicker.hour.isFocused()).to.equal(true);
+			// Step 10-2: Press 5-way Right.
+			await Page.spotlightRight();
+			await expect(await timePicker.minute.isFocused()).to.equal(true);
+			// Step 10-3: Press 5-way ok.
+			await Page.spotlightSelect();
+			// Step 10-2 Verify: Spotlight moves to the hour picker.
+			await expect(await timePicker.hour.isFocused()).to.equal(true);
+
+			// Step 11: 5-way ok on the hour picker.
+			await Page.spotlightSelect();
+			// Step 11 Verify: Spotlight moves on the meridiem picker.
+			await expect(await timePicker.meridiem.isFocused()).to.equal(true);
+
+			// Step 12: 5-way ok on the meridiem picker.
+			await Page.spotlightSelect();
+			// Step 12 Verify: Spotlight still on the meridiem picker.
+			await expect(await timePicker.meridiem.isFocused()).to.equal(true);
+		});
+	});
+
+	describe('es-ES locale', function () {
+		const timePicker = Page.components.timePickerDefault;
+
+		beforeEach(async function () {
+			await Page.open('?locale=es-ES');
+		});
+
+		it('should move to the next picker with 5-way OK key in es-ES locale', async function () {
```

Comment:
Would you please add [QWTC-2541] in this description?
```suggestion
		it('should move to the next picker with 5-way OK key in es-ES locale [QWTC-2541]', async function () {
```

Reply:
Suggested Changes.